### ISSUE TITLE
Attribute change on quick view page will not affect code in displayProductAdditionalInfo  hook

### ISSUE THREAD

**Issue by dheerajwebkul** [PrestaShop/PrestaShop#15287]
**Describe the bug**
On product page, if we change attribute, the core.js code updating the displayProductAdditionalInfo hook tpl code by using 'product-additional-info' class But this is not working on quick view page because on quick view tpl this hook is not wrap up in 'product-additional-info' class div
Check these code:

https://github.com/PrestaShop/PrestaShop/blob/1.7.6.1/themes/classic/templates/catalog/_partials/product-additional-info.tpl#L25

https://github.com/PrestaShop/PrestaShop/blob/1.7.6.1/themes/classic/templates/catalog/_partials/quickview.tpl#L74

**To Reproduce**
One way:
you need to create a module with hook 'displayProductAdditionalInfo' code and change this hook code by changing attribute with jquery.

Second way,
you can inspect in browser and see 'product-additional-info' class div code its changing only on product page not on quick view page because this class not found on quick view page.

**Additional information**
PrestaShop version: 1.7.6.0 and below
PHP version: 7

**Comment by khouloudbelguith** [comment]
Hi @dheerajwebkul,

Thanks for your report.
I manage to reproduce the issue with PS1.7.6.1 & PS1.7.5.2.
I used a module using this 
```
    public function hookDisplayProductAdditionalInfo($params)
    {
        $this->assignCombinations($params);
        return $this->display(__FILE__, 'test.tpl');
    }
```
a tpl file

```
    <div>
        {foreach from=$combinations item=combination}
                <section class="dfc_description_{$combination.id_product_attribute|intval}">
                    {$combination.description|escape:'quotes':'UTF-8' nofilter}
                </section>
        {/foreach}
    </div>
```
The 'product-additional-info' class But this is not working on quick view page 
https://drive.google.com/file/d/1fXiYXvMmdZJedBrx61yX6Q2sdi1XFaGK/view

Second way: I have the same issue.

I’ll add this to the debug roadmap.
Thanks!

**Comment by dheerajwebkul** [comment]
@khouloudbelguith yes thanks for the explanation.

I have fixed this in PR  #15297

Thanks,
